### ISSUE TITLE
Breadcrumb: Accessibility improvements

### ISSUE THREAD

**Issue by marcoscv-work** [liferay/clay#5441]
For a better component accessibility and operability we have designed and conceptualized a new concept for Breadcrumb:

**Requirements**:
- Component can be configured to collapse or not collapse items
- Remove truncate classes
- Add aria-current="page" to the current page link
- Add aria-controls="" and aria-expanded="" pattern to the "Expand Breadcrumb" button
- Add title to icon button for "See full nesting"
- Add link for all pages, even for current page
- All nested configuration should be deactivated from Accessibility menu, for this we add a general `no-nesting` class

Implementation example:
https://codepen.io/marcoscv/full/MWqvpRw

There is a bug when general class for expand the component is present:
<img width="347" alt="image" src="https://user-images.githubusercontent.com/7963804/227899427-5c522193-fa96-4ccd-bb17-d14bf62ca04c.png">
It should be:
<img width="356" alt="image" src="https://user-images.githubusercontent.com/7963804/227902050-baec66c2-e0ef-4d7a-8c66-539d035a8fb4.png">

**Comment by matuzalemsteles** [comment]
@marcoscv-work how is nested behavior? Is there any specific rule when it should be activated? For example, does it only make the last two items visible independent of quantity?

**Comment by marcoscv-work** [comment]
> 

- 4 items or more will be collapsed
- when collapsed we will show last two item, the parent one and the current (with link to the current page and `aria-current="page"`
